packages/shared/src/components/onboarding/PWA/useInstallPWA.ts

```diff
@@ -0,0 +1,42 @@
+import type { BrowserName } from '../../../lib/func';
+import { getCurrentBrowserName, isPWA } from '../../../lib/func';
+
+interface IBeforeInstallPromptEvent extends Event {
```

Comment:
We never really use the I prefix naming (non trivial but would avoid it moving forward)

Reply:
`BeforeInstallPromptEvent` is the original type name for this so I'm using a different name avoiding future collision. But I'm renaming in order to remove the prefix ✔️